doc/source/user/usage.rst

```diff
@@ -156,6 +159,33 @@ is called and passed the user-specified :class:`~watts.Parameters`::
 This will generate the OpenMC input files using the template parameters, run
 OpenMC, and collect the results.
 
+PyARC Plugin
+~~~~~~~~~~~~~
+
+The :class:`~watts.PluginPyARC` class handles PyARC execution in a similar
+manner to the :class:`~watts.PluginSAM` class for SAM. PyARC use text-based 
+input files which is templated as shown as follows:
+
+.. code-block:: jinja
+
+    surfaces{
+        hexagon ( hex ){ orientation=y   normal = z  pitch = {{ assembly_pitch }} }
+        plane ( z0 ) { z = 0.0  }
+        plane ( z10 ) { z = {{ assembly_length }} }
+    }
+
+If the templated input file is `pyarc_template`, then PyARC can be executed with the following command:
```

Comment:
It's not actually executing PyARC when we create the plugin, so we probably want to choose a different word than "executed"